README.md

```diff
@@ -511,8 +511,8 @@ const logger = createLogger('<YOUR KEY HERE>')
 logger.on('error', console.error)
 
 function onSignal(signal) {
-  logger.warn({signal}, 'received signal, shutting down')
```

Comment:
What would be the way to include this as metadata?

Reply:
`meta` is an object property of the `opts` parameter, so just add it there like this.

```js
logger.warn('received signal, shutting down', {meta: {signal}})
```
I can add that into the example if it's helpful.

Reply:
Oof, there were actually more changes that somehow didn't get saved and pushed up.  Fixing all.